.goreleaser.yml

```diff
@@ -10,7 +10,7 @@ builds:
     env:
       - CGO_ENABLED=0
     ldflags:
-      - -s -w -X github.com/railwayapp/cli/constants.Version={{.Version}}
+      - -s -w -X github.com/railwayapp/cli/constants.Version={{.Version}} -X github.com/railwayapp/cli/constants.RailwayURL={{"https://railway.app"}}
```

Comment:
Hmm, should this be `constants.RailwayURLDefault`? `RailwayURL` doesn't seem to exist

Reply:
Should be RAILWAY_URL

Will fix. RailwayURLDefault is left alone such that it can be the fallback